source/index.html.md

```diff
@@ -554,21 +554,29 @@ payment method must be omitted.
     <i>Optional when transaction is signed and partner is trusted.</i> <br />
     Card Security Code.
   </dd>
-  <dt>card[pares]</dt>
-  <dd>[:base64:] <br /> <i>Optional</i> <br /> See more information at <a target="_blank" href="http://docs.3dsecure.io">3Dsecure.io</a></dd>
+  <dt>card[3dsecure]</dt>
+  <dd>dictionary <br />
+    <i>Optional</i> <br />
+    See <a href="#authentication-3dsecure">Authentication: [3dsecure]</a>.
+  </dd>
+
+  <!-- deprecated -->
+  <dt><strike>card[pares]</strike></dt>
+  <dd>
+    Deprecated! Please use <code>card[3dsecure][v1][pares]</code>. <br />
+    [:base64:] <br />
+    <i>Optional</i> <br />
+    See more information at <a target="_blank" href="http://docs.3dsecure.io">3Dsecure.io</a>
+  </dd>
 </dl>
 
 <p class="alert alert-info">
-  <b>Notice:</b> A valid PARes included in <code>card[pares]</code> can indicate
-  3 different levels: non-authenticated, attempted 3-D Secure, fully 3-D Secure.
-  <br />
-  <b>Notice:</b> An authorization made with <code>card[]</code> is strongly
-  authenticated if it is fully 3-D Secure.
-  <br />
-  <b>Notice:</b> An authorization that includes <code>card[pares]</code> and/or
-  <code>card[csc]</code> cannot be a subsequent recurring authorization.
+  <b>Notice:</b> An authorization that includes
+  <code>card[3dsecure][v1][pares]</code>, <code>card[3dsecure][v2][rreq]</code>,
+  and/or <code>card[csc]</code> cannot be a subsequent recurring authorization.
```

Comment:
Not `card[3dsecure][v2][ares]`?

Reply:
Nope. It is possible to get an AV in the ARes for a subsequent recurring (thus, frictionless) transaction.